packages/web-components/fast-foundation/src/data-grid/data-grid-row.ts

```diff
@@ -355,7 +355,7 @@ export class FASTDataGridRow extends FASTElement {
                 : this.defaultHeaderCellItemTemplate;
     }
 
-    private updateRowStyle = (): void => {
+    protected updateRowStyle(): void {
```

Comment:
I think we should add a comment as this is becoming a public api.

Reply:
Ideas on what the comment should be? I'm thinking something like "Updates the row's `grid-template-columns` styles based on the column definitions of the parent data-grid."

Reply:
Or are you just referring to adding the `@public` tag to the method?

Reply:
Maybe that, and maybe rename it to a more narrow "applyGridTemplateColumns" or something like that?